GetOwnershipClaimStatus/__tests__/claimProcedureStatus.test.ts

```diff
@@ -0,0 +1,27 @@
+import { ClaimProcedureStatus } from "../../generated/definitions/ClaimProcedureStatus";
+import * as E from "fp-ts/lib/Either";
+
+const mockStatus = {
+  completed: 1,
+  failed: 0,
+  not_started: 0,
+  processing: 0
+};
+describe("ClaimProcedureStatus Type Check", () => {
+  it("should be a valid status with all fields with default value", () => {
+    const queryRes = ClaimProcedureStatus.decode({
+      completed: 1
+    });
+    if (E.isRight(queryRes)) {
+      expect(queryRes.right).toEqual(mockStatus);
+    }
+  });
```

Comment:
I'd separate test cases with an empty line to keep code blocks more visible. At first I thought that there was only 1 test.

Reply:
Yes, it's a good point. Thank you.

Reply:
Fix with 6260788